src/markdown-script.ts

```diff
@@ -1,14 +1,46 @@
-/**
```

Comment:
Why delete this comment 🥲 ?

Reply:
Whoops sorry about that haha, I will add it now